extensions/binding/org.eclipse.smarthome.binding.lifx/ESH-INF/i18n/lifx.properties

```diff
@@ -33,3 +33,7 @@ channel-type.lifx.temperaturezone.description = Sets the zone {0} temperature of
 # channel type configuration
 channel-type.config.lifx.brightness.powerOnBrightness.label = Power on brightness
 channel-type.config.lifx.brightness.powerOnBrightness.description = Brightness level used when switching on the light. Use empty value to leave brightness as is.
+channel-type.config.lifx.brightness.powerOnTemperature.label = Power on color temperature
+channel-type.config.lifx.brightness.powerOnTemperature.description = Color temperature used when switching on the light. Use empty value to leave color temperature as is.
+channel-type.config.lifx.brightness.powerOnColor.label = Power on color
+channel-type.config.lifx.brightness.powerOnColor.description = Color used when switching on the light. Use empty value to leave color as is.
```

Comment:
The keys in the properties file have to match the URI of the `config-description`. So I think it has to be:

```
channel-type.config.lifx.brightness.powerOnBrightness.label = Power on brightness
channel-type.config.lifx.brightness.powerOnBrightness.description = Brightness level used when switching on the light. Use empty value to leave brightness as is.
channel-type.config.lifx.color.powerOnBrightness.label = Power on brightness
channel-type.config.lifx.color.powerOnBrightness.description = Brightness level used when switching on the light. Use empty value to leave brightness as is.
channel-type.config.lifx.color.powerOnColor.label = Power on color
channel-type.config.lifx.color.powerOnColor.description = Color used when switching on the light. Use empty value to leave color as is.
channel-type.config.lifx.temperature.powerOnTemperature.label = Power on color temperature
channel-type.config.lifx.temperature.powerOnTemperature.description = Color temperature used when switching on the light. Use empty value to leave color temperature as is.
```